processing_r/processing/algorithm.py

```diff
@@ -334,8 +340,56 @@ def processAlgorithm(self, parameters, context, feedback):
                     f.write(RUtils.html_formatted_console_output(output))
                 self.results[RAlgorithm.R_CONSOLE_OUTPUT] = html_filename
 
+        if self.save_output_values and self.output_values_filename:
+            with open(self.output_values_filename, 'r') as f:
+                lines = [line.strip() for line in f]
+            # get output values stored into the file
+            outputs = self.parse_output_values(iter(lines))
+            # merge output values into results
+            for k, v in outputs.items():
+                if k not in self.results:
+                    self.results[k] = v
+
         return self.results
 
+    def parse_output_values(self, lines):
+        """
+        Parse the lines from an output values file
+        The lines are:
+        ## output1_name
+        output1_value
+        ## output2_name
+        output2_value
+        """
+        if not self.save_output_values:
+            return
+
+        outputs = {}
+        output = None
+        ender = 0
+        line = next(lines).strip('\n').strip('\r')
+        while ender < 10:
```

Comment:
Whats this supposed to do? break if there's 10 empty lines in a row? when would this happen?

Reply:
It is the same in `parse_script` and it was in the old plugin. The idea is to get multi-line values.

Some values can be multi-line and some line can be empty between 2 text lines.

Reply:
Ok, fair enough